src/main/java/com/microsoft/util/Utils.java

```diff
@@ -10,10 +10,7 @@
 import javax.lang.model.util.SimpleElementVisitor9;
 import javax.lang.model.util.Types;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;

```

Comment:
Why changing this?

Reply:
Since there is no collision for classes here, just to reduce the lines of import statement.

Reply:
Then I guess we can do the same for CommentHelper, don't we?

Reply:
It is said that single-type-import (such as _java.util.Collections_)can help increase the code compile. But noticed that, usually if the more than three types need to be imported,  _java.util.*_ will be used if there is no conflict, so just follow the convention.